.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "packages/.github"]
+	path = packages/.github
+	url = git@github.com:pulsar-edit/.github
```

Comment:
```suggestion
	url = https://github.com/pulsar-edit/.github
```
Wondering if there is an issue if we change this to https rather than SSH to make it easier for people to clone and build the docs without having SSH keys set up. Not an expert with submodules so happy to be told if this is incorrect.